functions/Autgrp/isometry.c

```diff
@@ -458,7 +458,8 @@ isometry (matrix_TYP **F1, matrix_TYP **F2, int Fanz, matrix_TYP *SV1, matrix_TY
 	}
 /* V.prime is a prime p, such that the entries of the short vectors remain 
    unchanged under reduction mod p in symmetric form, i.e. -p/2 < x <= p/2 */
-	for (V.prime = 2*max + 1; isprime(V.prime) == 0; ++V.prime);
+	for (V.prime = 2*max + 1; isprime(V.prime) == 0; ++V.prime)
+	    ;
```

Comment:
Why did you put the semicolon into a new line here?

Reply:
To indicate to the compiler that yes, an empty loop body is what we want; we didn't just by accident put a semicolon after the `for`.

An alternative would be to replace the semicolon by an empty block `{}`.

Reply:
Ah ok, that's fine by me then.